### PR TITLE
added usage for constants and some basic commands

### PR DESCRIPTION
**Pull request by slheavner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +65,28 @@ rokus: {
 }
 ```
 
+## Constants
+Each flavor can contain a `constants.yaml`. The strings defined in this file will be added to any `.xml` or `.brs` source files. You can identify the string based on its path in the yaml file. The constants file is idea behind the constants file is similar to Android resource files.
```
Comment (bret-fears) [inline README.md]: Might be nice to include a link to https://developer.android.com/guide/topics/resources/accessing-resources.html or something similar for folks who aren't familiar with how Android handles resources

Comment (nishtahir) [reply README.md]: I think we should explain how it works instead. Refering users to another system's documentation on how to use your tool is not the best.

**Comment by slheavner** [comment]
@nishtahir I'm not exactly sure what you are referring to and why you declined the PR? I am not referring users to another system's documentation, only stating that it's inspired by the Android resources. Furthermore, I have a full example of the constants file, a source file using strings from it, and the final output. Is that not explaining how it works?

**Comment by bret-fears** [comment]
@slheavner I think @nishtahir was rebutting my suggestion with the comment about referring users to other's documentation. Having an example in the constants is great and probably more than good enough.

**Comment by nishtahir** [comment]
Will finish this up

File: README.md [hunk 2]
```diff
@@ -66,22 +66,25 @@ rokus: {
...
+Each flavor can contain string resources specified in the `YAML` format by providing `constants.yaml` file. Strings can be referenced by their path specified in any `.xml` or `.brs` source files. For example,
 
-### Example
-constants.yaml
+Given a `constants.yaml` file: 
 ```
 strings:
   contactSupport: 'contact support at 555-555-5555'
-  login:
+  login: 'Hello Roku'
```
Comment (slheavner) [inline README.md]: no value for `login` here

**Pull request merged**